dms/views/directory.xml

```diff
@@ -0,0 +1,731 @@
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" />
+                    <field name="activity_ids" />
+                    <field name="message_ids" />
+                </div>
+            </form>
+        </field>
+    </record>
+    <record id="view_dms_directory_manager_form" model="ir.ui.view">
+        <field name="name">dms_directory.form</field>
+        <field name="model">dms.directory</field>
+        <field name="inherit_id" ref="view_dms_directory_form" />
+        <field name="arch" type="xml">
+            <field name="is_root_directory" position="after">
+                <field name="is_root_directory" groups="dms.group_dms_manager" />
+            </field>
+            <field name="storage_id" position="attributes">
+                <attribute name="options">{}</attribute>
+                <attribute name="groups">dms.group_dms_manager</attribute>
+            </field>
+        </field>
+    </record>
+    <record id="action_dms_directory" model="ir.actions.act_window">
+        <field name="name">Directories</field>
+        <field name="res_model">dms.directory</field>
+        <field name="view_mode">kanban,tree,form</field>
+        <field name="domain">[("is_hidden", "=", False)]</field>
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+                Add a new Directory.
+            </p>
+            <p>
+                Directories can be used to structure and organize
+                files directly in Odoo.
+            </p>
+        </field>
+    </record>
+    <menuitem
+        id="menu_dms_directory"
+        name="Directories"
+        sequence="30"
+        parent="main_menu_dms"
+        action="action_dms_directory"
+    />
+    <record id="view_dms_directory_new_form" model="ir.ui.view">
+        <field name="name">dms_directory.form</field>
+        <field name="model">dms.directory</field>
+        <field name="inherit_id" ref="view_dms_directory_form" />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//sheet//div[hasclass('oe_button_box')]" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//sheet//group[@name='settings']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//sheet//group[@name='data']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//sheet//group[@name='data']" position="after">
+                <group>
+                    <group>
+                        <field name="is_root_directory" readonly="1" />
```

Comment:
```suggestion
                        <field name="is_root_directory" readonly="1" groups="!dms.group_dms_manager" />
                        <field name="is_root_directory" groups="dms.group_dms_manager" />
```
The correct technique would be to add the field 2 times (each with the corresponding permissions).